modules/serverless-https-redirect-global.adoc

```diff
@@ -0,0 +1,35 @@
+// Module is included in the following assemblies:
+//
+// * serverless/admin_guide/knative-serving-CR-config.adoc
+
+[id="serverless-https-redirect-global_{context}"]
+= HTTPS redirection global settings
+
+You can enable HTTPS redirection for all services on the cluster, by configuring the `httpProtocol` spec for the `KnativeServing` custom resource, as shown in the following example:
+
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeServing
+metadata:
+  name: knative-serving
+spec:
+  config:
+    network:
+      httpProtocol: "enabled"
+...
+----
+
+Alternatively, you can set the `http-protocol` data field in the `config-network` config map:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-network
+  namespace: knative-serving
+data:
+  http-protocol: "enabled"
+...
+----
```

Comment:
We should not suggest that users modify `config-network` directory. It might be replaced by the config in `KnativeServing` above.
Let's remove this.

Reply:
Also the naming for `http-protocol` feels a bit disconnected with the the notion of redirection, might miss something here.